news/1562.bug

```diff
@@ -0,0 +1 @@
+Fix context navigation endpoint @contextnavigation to serve the navigation tree based on depth (bottomLevel).
```

Comment:
@andreiggr don't forget your username here for credits in the future please (will add it for you this time :)).

Reply:
Hi @tisto. Thanks for the pointers with workig on restapi!